composer.json

```diff
@@ -9,13 +9,14 @@
         }
     ],
     "require": {
```

Comment:
Nedavalo by tady smysl vypnout generovani `composer.lock`?
```suggestion
    "config": {
        "lock": false
    },
    "require": {
```

Reply:
ani nevím že taková option existuje nevím že bysme to někde používali.

Reply:
jj, treba na Storage clientovi a par dalsich: https://github.com/search?q=org%3Akeboola+%27%22lock%22%3A+false%27+filename%3Acomposer.json&type=code

Reply:
přidal sem